xbmc/GUIInfoManager.cpp

```diff
@@ -971,6 +971,14 @@ const infomap player_times[] =   {{ "seektime",         PLAYER_SEEKTIME },
 ///     @skinning_v17 **[New Infolabel]** \link Player_Process_videoheight `Player.Process(videoheight)`\endlink
 ///     <p>
 ///   }
+///   \table_row3{   <b>`Player.Process(videoscantype)`</b>,
+///                  \anchor Player_Process_videoscantype
+///                  _string_,
+///     @return The scan type (progressive / interlaced) of the currently playing video.
```

Comment:
You seem to return "p" and "i" in the infolabel implementation and not progressive/interlaced.

```suggestion
///     @return The scan type identifier of the currently playing video **p** (for progressive) or **i** (for interlaced).
```

Reply:
you are right so it is more understandable